### DISCUSSION TITLE
How to detect circular references in OpenApiSchema object

### DISCUSSION THREAD

**Discussion by gigadani** [microsoft/OpenAPI.NET#1031] [Q&A] [answered]
Hello,

As the title states, I'd like to know a nice and easy way to detect circular references in OpenApiSchema object.

Although I spent a hefty amount of time on googling this as well trying to RTFM for it, I failed to find an answer.

**Comment by baywet** [comment, marked as answer]
Hi @gigadani,

Thank you for using the SDK and for reaching out.

This is something we're adding to the library to be released soon.

https://github.com/microsoft/OpenAPI.NET/pull/3033

Let us know if you have any additional comments or questions.
